charts/opentelemetry-operator/templates/admission-webhooks/operator-webhook.yaml

```diff
@@ -133,8 +133,6 @@ webhooks:
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-  annotations:
-    cert-manager.io/inject-ca-from: {{ include "opentelemetry-operator.webhookCertAnnotation" . }}
```

Comment:
[Please note the sister mutating webhook](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/b8f1228a5718865adf840855e255dd840a282a90/charts/opentelemetry-operator/templates/admission-webhooks/operator-webhook.yaml#L31) in this same file specifically excludes the `cert-manager.io/inject-ca-from` annotation so cert-manager doesn't try to auto-inject a certificate it generated.